src/runtime/ControlHandler.ts

```diff
@@ -101,6 +101,9 @@ export class ControlHandler implements RequestHandler {
 
     constructor(controlManager: IControlManager, services?: ControlServicesProps) {
         this.controlManager = controlManager;
+        if (services !== undefined) {
+            ControlServices.setDefaults(services);
```

Comment:
same as for ControlManager... we should not make these kinds of of (global) side-effects

Reply:
Updated. 
Removed this block of code to set global services